src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="box">
+      <ul class="calendar calendar--start-day--sun calendar--month-lenght--29">
```

Comment:
```suggestion
      <ul class="calendar calendar--start-day--sun calendar--month-length--29">
```

Reply:
LOL, sorry for the dumb error. 
Fixed by now.